virttest/cpu.py

```diff
@@ -1258,3 +1261,132 @@ def get_qemu_best_cpu_model(params):
             return host_cpu_model
     # If no host cpu model can be found on qemu_cpu_models, choose the default
     return params.get("default_cpu_model", None)
+
+
+def get_cpu_info_from_cache(cache_file):
+    """
+    Get cpu info from cache_file
+    """
+    if os.path.exists(cache_file):
+        logging.info("Get cpu model and flags from cache file")
+        with open(cache_file) as f:
+            cpu_info = json.load(f)
+        return cpu_info
+    return
+
+
+def store_cpu_info_to_cache(cache_file, cpu_info):
+    """
+    Store cpu info into cache_file
+    """
+    with open(cache_file, "w") as f:
+        json.dump(cpu_info, f)
+    return
+
+
+def get_qemu_best_cpu_info(params):
+    """
+    Get the best cpu model and features available for qemu
+    """
+    cache_file = "qemu_best_cpu_cache"
+    cache_path = os.path.join(data_dir.get_tmp_dir(), cache_file)
+    cpu_info = get_cpu_info_from_cache(cache_path)
+    if not cpu_info:
+        cpu_info = {}
+        cpu_model = get_qemu_best_cpu_model(params)
+        cpu_features = ""
```

Comment:
How about add a short comment for this line to explain it is for better expansion in the future ? Someone may wonder why not combine it to L1299 directly.

Reply:
Good idea. Do you have any suggestion on the comment?

Reply:
`# Expend if needed`